data/json/recipes/recipe_others.json

```diff
@@ -3054,18 +3059,34 @@
     "qualities": [ { "id": "DIG", "level": 2 } ],
     "components": [ [ [ "bag_canvas", 1 ] ], [ [ "material_gravel", 2700 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "ACTIVE_EXERCISE",
+    "result": "anvil_crude",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 6,
+    "time": "20 h",
+    "autolearn": true,
+    "qualities": [ { "id": "HAMMER", "level": 2 }, { "id": "GRIND", "level": 2 }, { "id": "ANVIL", "level": 1 } ],
+    "tools": [ [ [ "crucible", -1 ], [ "crucible_clay", -1 ], [ "casting_mold", -1 ] ], [ [ "forge", 200 ], [ "oxy_torch", 200 ] ] ],
+    "proficiencies": [ { "proficiency": "prof_metalworking" }, { "proficiency": "prof_blacksmithing" } ],
+    "components": [ [ [ "steel_lump", 28 ] ] ]
+  },
   {
     "type": "recipe",
     "activity_level": "MODERATE_EXERCISE",
     "result": "anvil_crude",
+    "id_suffix": "angle_grind",
```

Comment:
Differentiating based on the rail stock is better than a tool it uses.

Reply:
Sure.